Binary_Search/RotationCount.java

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Incorrect folder name.

Reply:
Should I, then, put it under Array_Rotation or create a new folder? @jainaman224